pkg/gui/recent_repos_panel.go

```diff
@@ -60,8 +60,12 @@ func (gui *Gui) getCurrentBranch(path string) string {
 }
 
 func (gui *Gui) handleCreateRecentReposMenu() error {
-	// we skip the first one because we're currently in it
-	recentRepoPaths := gui.c.GetAppState().RecentRepos[1:]
+	// we'll show an empty panel if there are no recent repos
```

Comment:
random bug I found

Reply:
Do elaborate. Currently only the `cancel` button appears, what will happen in this case?

I tested it, `RecentRepos` always contains _at least one_ repo (the current one), and `[1:]` will result in an empty array in that case..

Reply:
when running the integration test we must have had zero recent repos there somehow. I'm aware that just showing the cancel button isn't great but this was just to prevent the app from crashing